src/Core/System/Snippet/Command/InstallTranslationCommand.php

```diff
@@ -40,10 +43,27 @@ protected function configure(): void
     protected function execute(InputInterface $input, OutputInterface $output): int
     {
         $locales = $this->getLocales($input);
-        $progressBar = $this->createProgressBar($output, \count($locales));
+        $metadata = $this->metadataLoader->getUpdatedMetadata($locales);
+
+        $localesRequiringUpdate = $metadata->getLocalesRequiringUpdate();
+        if ($localesRequiringUpdate === []) {
+            $output->writeln('All translations are up to date.');
+
+            return self::SUCCESS;
+        }
+
+        $localesDiff = array_diff($locales, $localesRequiringUpdate);
+        if ($localesDiff !== []) {
```

Comment:
Is there a reason you compare with an empty array? I'd rather use "empty" or "count > 0" or something like that. Pure curiosity.